src/select.jsx

```diff
@@ -197,8 +199,12 @@ var classBase = React.createClass({
     });
   },
   onBlur () {
+    var self = this;
+
     this.setState({
       focus: false
+    }, function () {
```

Comment:
Just use `() => { this.props.onBlur }` instead of `self = this`. Arrow function keep the current context AKA just get babelified to `bind(this)`.